docs/streams/developer-guide/dsl-api.html

```diff
@@ -58,13 +58,15 @@
                             <li><a class="reference internal" href="#hopping-time-windows" id="id21">Hopping time windows</a></li>
                             <li><a class="reference internal" href="#sliding-time-windows" id="id22">Sliding time windows</a></li>
                             <li><a class="reference internal" href="#session-windows" id="id23">Session Windows</a></li>
+                            <li><a class="reference internal" href="#window-final-results" id="id31">Window Final Results</a></li>
```

Comment:
I'm wondering if it's simpler to make the ids the same as the `href` just without the `#`, but definitely _**not**_ for this PR, just a thought off the top of my head

Reply:
I agree in spirit with @bbejeck . If we don't think that just "being careful" is a good strategy (which seems reasonable), then, rather than re-numbering the list every time we add something (remember Basic?), we should just use named identifiers.

We can't make the id the same as the one we're linking to (which is what is in the href property), but we can give it some kind of reasonable related name (like `toc-window-final-results`).

It would make life a little easier for sure.

Reply:
SGTM. Input from @JimGalasyn and @guozhangwang would be great here.